README.md

```diff
@@ -48,9 +48,7 @@ $ ./_output/bin/descheduler --help
 Descheduler can be run as a job inside of a pod. It has the advantage of
 being able to be run multiple times without needing user intervention.
 Descheduler pod is run as a critical pod to avoid being evicted by itself,
-or by kubelet due to an eviction event. Since critical pods are created in
-`kube-system` namespace, descheduler job and its pod will also be created
-in `kube-system` namespace.
+or by kubelet due to an eviction event.
```

Comment:
As of now, critical pods are limited to `kube-system` namespace. I think having another annotation specific to descheduler to define critical pods may confuse end-users. I am working on ensuring that critical pods could be created in any namespace that admin wants to be in using quota(https://github.com/kubernetes/kubernetes/pull/76310). I will try to get to it sometime next week. After that, I think this problem should be solved too.